docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,574 @@
+
+A more flexible approach is to prepare various JSON stylesheets and request a different style for each platform. The [Tizen.NUI.NUIApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIApplication.html) class has constructors that take a stylesheet argument:
+
+```csharp
+class Example : NUIApplication
+
+Example example = new Example("example-path/example.json");
+```
+
+To change the font style for standard text controls, use the following JSON syntax:
+
+```csharp
+{
+   "styles":
+   {
+      "textlabel":
+      {
+         "fontFamily": "FreeSerif",
+         "fontStyle":
+         {
+            "weight": "bold",
+            "slant": "italic"
+         },
+         "pointSize": 8
+      }
+   }
+}
+```
+
+However, the same `pointSize` is unlikely to be suitable for all text controls in an application. To define custom styles for existing controls, set a style name for each case, and provide a style override in a JSON stylesheet.
+
+You can provide further flexibility for the various screens by mapping the logical size to a physical size in the stylesheet.
+
+<a name="align"></a>
+### Align Text
+
+To align the text in a text label:
+
+-   To enable text wrapping, use the `MultiLine` property of the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class:
+
+    ```csharp
+    label.MultiLine = true;
+    ```
+
+-   To align the text horizontally to the beginning, center, or end of the available area, set the `HorizontalAlignment` property of the `Tizen.NUI.BaseComponents.TextLabel` class with the corresponding value of the [Tizen.NUI.HorizontalAlignment](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.HorizontalAlignment.html) enumeration:
+
+    ```csharp
+    label.HorizontalAlignment = HorizontalAlignment.Begin;
+    label.HorizontalAlignment = HorizontalAlignment.Center;
+    label.HorizontalAlignment = HorizontalAlignment.End;
+    ```
+
+    The following table summarizes the available values of the `Tizen.NUI.HorizontalAlignment` enumeration for both left-to-right (Latin) and right-to-left (Arabic) script. In addition, for the illustrated examples, it is assumed that the label size is greater than the minimum required size:
+
+    |  Alignment  | Left-to-right script example    |      Right-to-left script example |
+    |-------------|---------------------------------|-----------------------------------|
+    |  `Begin` |   ![Latin script aligned to beginning](media/LatinBegin.png) |  ![Arabic script aligned to beginning](media/ArabicBegin.png) |
+    |`Center` |  ![Latin script aligned to center](media/LatinCenter.png) | ![Arabic script aligned to center](media/ArabicCenter.png) |
+    | `End`   |  ![Latin script aligned to end](media/LatinEnd.png) |   ![Arabic script aligned to end](media/ArabicEnd.png) |
+
+
+
+<a name="decorations"></a>
+### Use Decorations

```

Comment:
### Use Decorations for Text

Reply:
Referencing https://github.com/Samsung/tizen-docs/pull/1009#discussion_r398493508
I add the updated suggestion

\### Use Decorations for TextLabel